go.mod

```diff
@@ -1,27 +1,90 @@
 module github.com/codingben/kubevirt-disk-uploader
 
-go 1.22.4
+go 1.22.2
+
+replace (
+	// Fixed to versions v0.26.x for kubevirt.io/client-go v1.2.x
+	k8s.io/api => k8s.io/api v0.26.15
+	k8s.io/client-go => k8s.io/client-go v0.26.15
+)
 
 require (
 	github.com/google/go-containerregistry v0.20.2
 	github.com/spf13/cobra v1.8.1
+	kubevirt.io/client-go v1.2.2
```

Comment:
please use version  v1.3.0

Reply:
I really struggled to import kubevirt's client-go, I had lots of depednencies issues. I'll try to update it to v1.3.0.